pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistryServiceImpl.java

```diff
@@ -467,4 +493,12 @@ static SchemaData schemaInfoToSchema(SchemaRegistryFormat.SchemaInfo info) {
         }
     }
 
+    public static boolean isUsingAvroSchemaParser(SchemaType type) {
+        if (type == SchemaType.AVRO || type == SchemaType.PROTOBUF || type == SchemaType.JSON) {
```

Comment:
nit:
you can use a `switch` block

Reply:
oh yes, I will change it to switch block.